.github/workflows/ci.yaml

```diff
@@ -0,0 +1,53 @@
+name: ci
```

Comment:
Change filename to build.yaml & add file dev.build.yaml to run on unmerged branches